<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainViewClass</class>
 <widget class="QMainWindow" name="MainViewClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Night Owl Trader</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,10,4">
      <item>
       <layout class="QVBoxLayout" name="orderLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Orders</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
        <item>
         <layout class="QVBoxLayout" name="controlLayout">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Commands</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="graphLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Chart</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="transactionsLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Transaction History</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="statusLabel">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>24</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 112, 67);
color: rgb(48, 48, 48);</string>
      </property>
      <property name="text">
       <string>Status: Inactive...</string>
      </property>
      <property name="margin">
       <number>10</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
